addons/web/tests/test_click_everywhere.py

```diff
@@ -6,7 +6,7 @@
 from datetime import datetime
 from odoo.addons.base.tests.common import HttpCaseWithUserDemo
 from dateutil.relativedelta import relativedelta
-
+from odoo.addons.account.tests.common import AccountTestMockOnlineSyncCommon
```

Comment:
We can't, web does not depend on account

Reply:
I know, it's just for testing